pkgs/applications/misc/joplin-desktop/default.nix

```diff
@@ -2,7 +2,7 @@
 
 let
   pname = "joplin-desktop";
-  version = "1.8.5";
+  version = "2.1.8";
```

Comment:
https://github.com/laurent22/joplin/releases/tag/v2.1.9

Reply:
Sorry for the off-by-one :)